wandb/sdk/data_types.py

```diff
@@ -857,16 +861,26 @@ def seq_to_json(
 
 class Molecule(BatchableMedia):
     """
-    Wandb class for Molecular data
+    Wandb class for 3D Molecular data
 
     Arguments:
         data_or_path: (string, io)
             Molecule can be initialized from a file name or an io object.
     """
 
-    SUPPORTED_TYPES = set(
-        ["pdb", "pqr", "mmcif", "mcif", "cif", "sdf", "sd", "gro", "mol2", "mmtf"]
-    )
+    SUPPORTED_TYPES = {
+        "pdb",
+        "pqr",
+        "mmcif",
+        "mcif",
+        "cif",
+        "sdf",
+        "sd",
+        "gro",
+        "mol2",
+        "mmtf",
+    }
+    SUPPORTED_RDKIT_TYPES = {"mol", "mol2", "pdb", "png", "tpl", "sdf"}
```

Comment:
we can turn an image into a molecule file? how does that work?!

Reply:
`rdkit` knows how to store the structural info about the molecule in the metadata so that it can reconstruct the original, see [here](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolfiles.html#rdkit.Chem.rdmolfiles.MolFromPNGFile). Pretty cool :)